static/js/src/advantage/contracts-api.js

```diff
@@ -222,3 +222,24 @@ export async function postCustomerInfoForPurchasePreview(
   let data = await response.json();
   return data;
 }
+
+export async function setAutoRenewal(subscriptionId, value) {
+  const queryString = window.location.search;
```

Comment:
I assume this is just for `test_backend` maybe a comment here would make to clearer in the future

Reply:
I added this as way to pass query params to the backend. It's currently only used for `test_backend` but could possibly be used for more. I'll add a comment